@ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/WebsiteJsonLd/test.js

```diff
@@ -1,23 +1,14 @@
 import React from 'react'
 import renderer from 'react-test-renderer'
 import WebsiteJsonLd from './'
-import { JsonLd } from 'gatsby-plugin-next-seo'
 
 describe('WebsiteJsonLd component', () => {
 
   it('renders props into values', () => {
-    const tree = renderer.create(<WebsiteJsonLd pathname='/' siteUrl='https://marble.nd.edu' />)
-    const testInstance = tree.root
+    const tree = renderer
+      .create(<WebsiteJsonLd pathname='/' siteUrl='https://marble.nd.edu' />)
+      .toJSON()
 
-    expect(testInstance.findByType(JsonLd).props.url).toBe('https://marble.nd.edu')
-    expect(testInstance.findByType(JsonLd).props.context).toBe('https://schema.org')
-    expect(testInstance.findByType(JsonLd).props.potentialAction.target).toBe('https://marble.nd.edu/search?q={search_term_string}')
-    expect(testInstance.findByType(JsonLd).props.potentialAction.type).toBe('SearchAction')
-    expect(testInstance.findByType(JsonLd).props.potentialAction.queryInput).toBe('required name=search_term_string')
-  })
-
-  it('renders on homepage', () => {
```

Comment:
this test is still required even in the snapshoting method because it is expected to return no results

Reply:
returned check for null